frontend/mms-Admin/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "app",
   "version": "0.1.0",
```

Comment:
Please, revert package-lock back. upgrade your note to use Node 16+ (18 recommended)
do not have a package lock change when they is no package.json changes

Reply:
I didn't make any changes tho
I'll resolve it.

Reply:
There's been a lot of changes since we reconfigured the project setup

Reply:
I'll reverted the package-lock file but we still need to update it later